src/Interfaces/Options/Particles/IOptionsParticlesOpacityAnimation.ts

```diff
@@ -0,0 +1,6 @@
+export interface IOptionsParticlesOpacityAnimation {
+    enable: boolean;
+    speed: number;
+    opacity_min: number;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'opacity_min' is not in camel case.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5098130)